policyengine_us/variables/gov/states/nj/njdhs/tanf/income/nj_tanf_countable_earned_income.py

```diff
@@ -0,0 +1,75 @@
+        months_enrolled_in_tanf = person("months_enrolled_in_tanf", period)
+        weekly_hours_worked = person("weekly_hours_worked", period)
+        # New Jerset Admin Code 10:90-3.8(b)
+        person_meet_higher_work_hours_threshold = (
+            weekly_hours_worked >= p.work_hours_threshold
+        )
+        person_meet_lower_work_hours_threshold = (
+            weekly_hours_worked < p.work_hours_threshold
+        )
+        person_enrolled_in_tanf_for_first_month = (
+            months_enrolled_in_tanf <= p.first_month_threshold
+        )
+        person_enrolled_in_tanf_for_consecutive_months_with_work_hours_over_20 = (
+            months_enrolled_in_tanf <= p.consecutive_month_threshold
+            and months_enrolled_in_tanf > p.first_month_threshold
+            and person_meet_higher_work_hours_threshold
+        )
+        person_enrolled_in_tanf_for_additional_months_with_work_hours_over_20 = (
+            months_enrolled_in_tanf > p.consecutive_month_threshold
+            and person_meet_higher_work_hours_threshold
+        )
+        person_enrolled_in_tanf_for_additional_months_with_work_hours_below_20 = (
+            months_enrolled_in_tanf > p.first_month_threshold
+            and person_meet_lower_work_hours_threshold
+        )
+        # The value of result will only change, if the first condition in "where" is met. If the condition is not met, the value of result will not change
+        result = 0
+        # If applicant is enrolled in TANF for the first month, the earned income deduction is 100%.
+        result = where(
+            person_enrolled_in_tanf_for_first_month,
+            gross_earned_income
+            * (1 - p.higher_work_hours.first_month_percent),
+            result,
+        )
+        # If applicant is enrolled in TANF for consecutive months with work hours over 20, the earned income deduction is 75%.
+        result = where(
+            (
+                person_meet_higher_work_hours_threshold
+                and person_enrolled_in_tanf_for_consecutive_months_with_work_hours_over_20
+            ),
+            gross_earned_income
+            * (1 - p.higher_work_hours.consecutive_month_percent),
+            result,
+        )
+        # If applicant is enrolled in TANF for additional months with work hours over 20, the earned income deduction is 50%.
+        result = where(
+            person_enrolled_in_tanf_for_additional_months_with_work_hours_over_20,
+            gross_earned_income * (1 - p.higher_work_hours.additional_percent),
+            result,
+        )
+        # If applicant is enrolled in TANF for additional months with work hours below 20, the earned income deduction is 50%.
+        result = where(
+            person_enrolled_in_tanf_for_additional_months_with_work_hours_below_20,
+            gross_earned_income * (1 - p.lower_work_hours.additional_percent),
+            result,
+        )
+        return result
```

Comment:
Here's an example of how to use `select()`:
https://github.com/PolicyEngine/policyengine-us/blob/83b0d38b209f07dbe2b7d593c3e7dce7e26778b6/policyengine_us/variables/gov/states/nj/tax/income/nj_main_income_tax.py#L4

The function's first argument is a condition list, the second argument is the list of values to return if that condition is true, and the (optional) third argument is the value to return if none of them are true (by default this is zero). The function goes through the condition list and returns the corresponding value. For example, if the third condition is true, the third value in the return list would be returned.
Here, you would want something like
`select( [ enrolled_in_tanf_first_month, meets_hours_threshold, ... ],
               [ gross_earned_income * ( 1-p.higher_work_hours.first_month_percent),
                 gross_earned_income
            * (1 - p.higher_work_hours.consecutive_month_percent),
            ... ]
)
`
You don't need to specify a default return value since you want zero anyway.